va/caa.go

```diff
@@ -104,6 +104,12 @@ func filterCAA(rrs []*dns.CAA) ([]*dns.CAA, []*dns.CAA, bool) {
 			issue = append(issue, caaRecord)
 		case "issuewild":
 			issuewild = append(issuewild, caaRecord)
+		case "iodef":
+			// We support the iodef property tag inasumch as we recognize it, but we
```

Comment:
Typo.

Reply:
Why the heck does codespell claim that "EDE" is a typo but doesn't catch "inasumch"